src/main/webapp/WEB-INF/views/creationForm.jsp

```diff
@@ -11,15 +11,15 @@
   <center>
 
     <h1>
-      <form action="process.jsp">
+      <form action="/DeveloperApi/form" method="POST">
 
-        <input type="text" name="firstName" value="First Name..." onclick="this.value=''"/><br/>
+        <input type="text" name="firstname" value="First Name..." onclick="this.value=''"/><br/>
 
-        <input type="text" name="lastName"  value="Last Name..." onclick="this.value=''"/><br/>
+        <input type="text" name="lastname"  value="Last Name..." onclick="this.value=''"/><br/>
 
         <input type="text" name="age"  value="Age..." onclick="this.value=''"/><br/>
 
-        <input type="text" name="programmingLanguage" value="Programming Language..." onclick="this.value=''"/><br/>
+        <input type="text" name="proglang" value="Programming Language..." onclick="this.value=''"/><br/>
```

Comment:
Rename this field to `programmingLanguage` in the model and services as well.

Reply:
fixed